02-express-mongo-crud/mongoose-intro/27-28/README.md

```diff
@@ -0,0 +1,252 @@
+# <img src="https://cloud.githubusercontent.com/assets/7833470/10423298/ea833a68-7079-11e5-84f8-0a925ab96893.png" width="60">  Mongo & Mongoose
+| Objectives |
+| :--- |
+| Traverse a MongoDB using the command line interface (CLI). |
+| Connect to MongoDB using Mongoose within your application. |
+
+## Review: What are Mongo and Mongoose?
+
+<img src="http://i.imgur.com/MVaVKG8.gif" />
+
+There are two main categories of databases: relational (SQL) databases, and non-relational (no-SQL) databases. Mongo is a no-SQL database that stores entries in a JSON-like format.
```

Comment:
Considering removing this section.  As we've discussed students don't have a basis for this comparison so it might make more sense when we hit SQL.